projects/ui/option/option.ts

```diff
@@ -119,6 +127,14 @@ export class CkOption<T = unknown> implements Highlightable, FocusableOption {
     this.selectionChange.emit(new CkOptionSelectionChange<T>(this))
   }
 
+  public toggle(emitEvent = true): void {
+    if (this.isSelected()) {
+      this.deselect(emitEvent)
+    } else {
+      this.select(emitEvent)
+    }
```

Comment:
this could be 2 lines xD

Reply:
Yes, I did it but got an eslint error `Expected an assignment or function call and instead saw an expression`

Reply:
This way shouldn't get that
```ts
    if (this.isSelected()) this.deselect(emitEvent)
    else this.select(emitEvent)
```

Reply:
Done